README.md

```diff
@@ -56,12 +56,14 @@ The script will run from start to finish in under 2 minutes, giving you an easy
   1. Ensure that your system is setup to run `oc cluster up`
       * Follow [these instructions](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) if you haven't already seen success with `oc cluster up`
   1. Download [run_latest_build.sh](https://raw.githubusercontent.com/openshift/ansible-service-broker/master/scripts/run_latest_build.sh)
-
-              wget https://raw.githubusercontent.com/openshift/ansible-service-broker/master/scripts/run_latest_build.sh
-              chmod +x ./run_latest_build.sh
-
+      ```
+      wget https://raw.githubusercontent.com/openshift/ansible-service-broker/master/scripts/run_latest_build.sh
+      chmod +x ./run_latest_build.sh
+      ```
```

Comment:
+1 definitely breaking out the commands into pre-formatted blocks makes them clearer.